ui/open-api-ui/src/main/java/io/smallrye/openapi/ui/IndexHtmlCreator.java

```diff
@@ -305,6 +305,7 @@ private static void addUrlSection(Map<String, String> urls, String urlsPrimaryNa
         DEFAULT_OPTIONS.put(Option.tagsSorter, null);
         DEFAULT_OPTIONS.put(Option.onComplete, null);
         DEFAULT_OPTIONS.put(Option.syntaxHighlight, null);
+        DEFAULT_OPTIONS.put(Option.queryConfigEnabled, "false");
```

Comment:
Probably you need to use null for default value ?

Reply:
I will change for null DEFAULT_OPTIONS.put(Option.queryConfigEnabled, "null"); its ok?

Reply:
I think it needs to be `DEFAULT_OPTIONS.put(Option.queryConfigEnabled, null);`